CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## Version 3.14.0
+
+- #277 - Add support for DOKS clusters as peers in Firewall rules
```

Comment:
```suggestion
- #277 - @caiofilipini - Add support for DOKS clusters as peers in Firewall rules
```

We've tended to include the contributor name in these. We should probably just standardize on the format that  `github-changelog-generator -org digitalocean -repo droplet_kit` produces.

Reply:
thanks andrew!